src/devices/Arduino/ConfigurableFirmata/ConfigurableFirmata.ino

```diff
@@ -0,0 +1,223 @@
+/*
+ * CommonFirmataFeatures.ino generated by FirmataBuilder
```

Comment:
I assume this isn't part of the project. Make sure this is highly visible in the docs. Consider moving this into sample.

@richlander does externally generated file need to have copyright header?

Reply:
This file (or some adaption of it) is used to load the arduino. If the Arduino IDE is installed, double clicking on this file opens up the IDE and directly allows flashing the firmware. It needs to be in a folder with the same name. 

I'll check again that this process is correctly documented.

Reply:
I understand, what I was asking if the more appropriate place to put this would be samples code, it's more likely to be noticed in there IMO

Reply:
It's linked to in the Readme, so I hope it can be found.